### DISCUSSION TITLE
Error message during build process

### DISCUSSION THREAD

**Discussion by js-midi** [shanteacontrols/OpenDeck#108] [Q&A]
This is the error message you see while following the wiki exampl
![스크린샷(1)](https://user-images.githubusercontent.com/80445117/134751977-d3b9d30f-2cb9-43cc-80c2-120d225805eb.png)
![스크린샷(2)](https://user-images.githubusercontent.com/80445117/134751988-f2f55b22-fa88-46e0-87cd-c45b578fe5b3.png)
e.
help me please

**Comment by js-midi** [comment]
Please review this error message
stm32 series boards build normally
but mega2560 shows this message when build
thanks
![스크린샷(1)](https://user-images.githubusercontent.com/80445117/135756929-23ff04b3-d106-4069-bf2e-86fac053d8db.png)

**Reply by js-midi** [reply]
It's really difficult to use Linux if you're not used to it.
Unknown error messages annoy me

**Comment by paradajz** [comment]
Windows annoys _me_. Therefore, I have decided to stop supporting it for flashing AVR boards. Wiki has been edited to reflect that. Do it either on Linux or macOS, or just keep using STM32 boards for which I support flashing on Windows.
